packages/x-charts-pro/tsconfig.json

```diff
@@ -8,7 +8,8 @@
       "mocha",
       "node"
     ],
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "skipLibCheck": true
```

Comment:
Added these to avoid `@emotion` errors.
We should be able to remove them once `@emotion` adds support for React 19.
Reference: https://github.com/emotion-js/emotion/pull/3291, https://github.com/emotion-js/emotion/issues/3271